### ISSUE TITLE
Pencil does not work anymore with newest firefox version

### ISSUE THREAD

**Issue by ManuelB** [prikhi/pencil#784]
```
$ pencil
Error: Platform version '43.0' is not compatible with
minVersion >= 36.0
maxVersion <= 42.*
```

Quick workaround:

sudo vi /usr/share/pencil/application.ini

```
[App]
Name=Pencil
Version=2.0.14.20150921
BuildID=20150921
Copyright=Copyright (c) 2015 Pavan Rikhi. Base on code copyright 2009-2013 Evolus.
ID=pencil@evolus.vn

[Gecko]
MinVersion=36.0
MaxVersion=43.*
```

**Comment by ManuelB** [comment]
Sorry, I used an old version 2.0.14. The new one is compatible.

**Comment by ManuelB** [comment]
Just double checked in master it is fixed but 2.0.15 will still have this problem.
